src/main/java/org/broadinstitute/hellbender/tools/spark/sv/StringKmerizer.java

```diff
@@ -0,0 +1,76 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import java.util.Iterator;
+import java.util.NoSuchElementException;
+
+/**
+ * Iterator over successive Kmers from a String.
+ * Silently skips over parts of the sequence that has characters other than A, C, G, or T.
+ *
+ * Created by tsharpe on 12/10/15.
+ */
+public final class StringKmerizer implements Iterator<Kmer>
```

Comment:
you probably want explicit tests for toString since it's strangely the most complicated method in the class